functions/graphql/schema/directives/auth.js

```diff
@@ -73,9 +73,8 @@ class AuthDirective extends SchemaDirectiveVisitor {
         // Get the required Role from the field first, falling back
         // to the objectType if no Role is required by the field:
         // Grab the user off of context
-        const [data, queryArgs, context] = args;
+        const [data = {}, queryArgs, context] = args;
```

Comment:
Does the data = {} mean that if the data key is not found in the args object, it will default to being an empty object?

Reply:
Yes.